docs/modules/ROOT/assets/images/fire_riskpool_registration.md

```diff
@@ -0,0 +1,24 @@
+# Fire riskpool registration
+
+```mermaid
+sequenceDiagram
+    actor RK as RiskpoolKeeper
+    actor IO as InstanceOperator
+    participant G as GIF
+    participant R as FireRiskpool
+    participant RW as RiskpoolWallet
+    
+    
+    IO ->> G: grant role to RiskpoolKeeper
+    RK ->> R: deploy FireRiskpool contract
+    RK ->>+ G: propose FireRiskpool
+    G ->>- R: after propose callback
+    R ->> G: register FireRiskpool
+    IO ->>+ G: approve FireRiskpool
+    G -->>- R: after approve callback
+    RK ->>+ R: set maximum number of active bundles
+    R ->>- G: set maximum number of active bundles
+    IO ->>+ G: set RiskpoolWallet for FireRiskpool
+    RW -->> R: GIF assigns RiskpoolWallet to FireRiskpool 
```

Comment:
There is not active functon call from the wallet to the riskpool here so. Please remove this and then also the RiskpoolWallet is not necessary as a participant.